google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanelTest.java

```diff
@@ -322,6 +325,90 @@ public void getSelectedLibraries_withSomeSelected_returnsLibraries() {
     assertThat(panel.getSelectedLibraries()).containsExactly(library1, library2);
   }
 
+  @Test
+  public void getApisToEnable_shouldEnableSelectedLibraryByDefault() {
+    CloudLibrary library = LIBRARY_1.toCloudLibrary();
+
+    GoogleCloudApiSelectorPanel panel =
+        new GoogleCloudApiSelectorPanel(ImmutableList.of(library), testFixture.getProject());
+    JTable table = panel.getCloudLibrariesTable();
+    checkCheckbox(table, 0);
+
+    Set<CloudLibrary> apisToEnable = panel.getApisToEnable();
+    assertThat(apisToEnable).containsExactly(library);
```

Comment:
This test depends on value of that `SHOULD_ENABLE_API_DEFAULT` constant, correct?

Reply:
yes